### PR TITLE
Spelling: Could not, authenticated, …

### PR DESCRIPTION
**Pull request by comradekingu**
The [translation platform](https://translate.ubports.com/translate/ubports/sync-monitor/nb/?checksum=945b580887c2979f) somehow has
https://github.com/ubports/sync-monitor/blob/8dfcae07341f969deb6dc536e7f5fcd1dec05278/authenticator/main.qml#L126
in it

### CODE REVIEW

**Comment by UniversalSuperBox** [comment]
Hi @comradekingu,

It appears you found a bug with this app using UTF-8 in its translation sources. Good work! :D

I've fixed the issue and also regenerated the translation sources using `make sync-monitor.pot` in its source tree. If this looks good to you, I'll approve and merge it.

File: po/sync-monitor.pot [hunk 1]
```diff
@@ -1,115 +1,150 @@
...
-#: ../src/sync-account.cpp:757
-msgid "Disk full"
+#: src/sync-account.cpp:772
+msgid "Connection certificate is invalid"
 msgstr ""
 
-#: ../src/sync-account.cpp:759
-msgid "Failed to sync due to a remote problem"
+#: src/sync-account.cpp:768
+msgid "Connection timeout"
```
Comment (comradekingu) [inline po/sync-monitor.pot:45]:
```suggestion
msgid "Connection timed out"
```
Avoids this issue https://grammarist.com/spelling/timeout-time-out/
https://www.dictionary.com/browse/timeout

File: po/sync-monitor.pot [hunk 2]
```diff
@@ -1,115 +1,150 @@
...
 
-#: ../src/sync-account.cpp:752
+#: src/sync-account.cpp:748
 msgid "Command not allowed"
 msgstr ""
 
-#: ../src/sync-account.cpp:755
-msgid "Proxy authentication required"
+#: src/sync-account.cpp:770
+msgid "Connection certificate has expired"
```
Comment (comradekingu) [inline po/sync-monitor.pot:37]:
```suggestion
msgid "Expired connection certificate"
```

File: po/sync-monitor.pot [hunk 3]
```diff
@@ -1,115 +1,150 @@
...
-#: ../src/sync-account.cpp:755
-msgid "Proxy authentication required"
+#: src/sync-account.cpp:770
+msgid "Expired connection certificate"
 msgstr ""
 
-#: ../src/sync-account.cpp:757
-msgid "Disk full"
+#: src/sync-account.cpp:772
+msgid "Connection certificate is invalid"
```
Comment (comradekingu) [inline po/sync-monitor.pot:41]:
```suggestion
msgid "Invalid connection certificate"
```

**Comment by comradekingu** [comment]
@UniversalSuperBox Top stuff. Looks good to me.
Found some (additional) minor savings.

**Review by UniversalSuperBox — APPROVED** [review]

**Pull request merged**